spec/classes/redmine/redmine_spec.rb

```diff
@@ -93,6 +93,48 @@
 
   end
 
+  context 'provider install' do
+    let :params do
+      {
+        :provider => 'svn'
+      }
+    end
+
+    it {should contain_package('subversion')}
+    it { should contain_vcsrepo('redmine_source').that_requires('Package[subversion]')  }
+  end
+
+  context 'autodetect mysql adapter' do
+    context 'mysql2' do
+      let :facts do
+        {
+          :osfamily                   => 'Redhat',
+          :operatingsystemrelease     => '6',
+          :operatingsystemmajrelease  => '6',
+          :domain                     => 'test.com',
+          :concat_basedir             => '/dne',
+          :rubyversion                => '1.9',
```

Comment:
How about a test to make sure we're doing the right thing with a higher version of ruby too?